exercises/concept/vehicle-purchase/.docs/instructions.md

```diff
@@ -0,0 +1,56 @@
+# Instructions
+
+In this exercise, you will write some code to help you prepare to buy a vehicle.
+
+You have three tasks, one to determine if you will need to get a licence, one to help you choose between two vehicles, and one to estimate the acceptable price for a used vehicle.
+
+## 1. Determine if you will need a drivers licence
+
+Some kinds of vehicles require a drivers license to operate them.
+Assume only the kinds `"car"` and `"truck"` require a license, everything else can be operated without a license.
+
+Implement the `needs_license` function that takes the kind of vehicle and returns a boolean indicating whether you need a license for that kind of vehicle.
+
+```jq
+"car" | needs_license
+# => true
+
+"bike" | needs_license
+# => false
+```
+
+## 2. Choose between two potential vehicles to buy
+
+You evaluate your options of available vehicles.
```

Comment:
Do you want to use past tense here? Present tense makes it sound like it might be "you need to do this" vs "this is story background". The next section has past tense in the story.

Reply:
yes, it reads better. thanks